src-files/def-config.yaml

```diff
@@ -48,6 +48,9 @@ verification:
   # wait this amount of time before attempting to verify the link again.
   defaultRetryAfter: 30s
 
+  # Count of attempts of getting access to remote data (links to the sites, etc)
+  maxRetries: 20
```

Comment:
The hardcoded value of 20 was very, very conservative. Now that this isn't hardcoded anymore, maybe 3 would be a more sensible default?